File: 7-kyu/turn-any-word-into-a-beef-taco/javascript/solution.js
const tacofy = (word) => {
  const ingredients = {"a": "beef", "e": "beef", "i": "beef", "o": "beef", "u": "beef",
                       "t": "tomato", "l": "lettuce", "c": "cheese", "g": "guacamole", "s": "salsa"};
  let ret = ["shell"];
  const insides = word.split("").filter((elem) => ingredients.hasOwnProperty(elem.toLowerCase())).map((ch) => ingredients[ch.toLowerCase()]);
  ret = ret.concat(insides);
  ret.push("shell");
  return ret;
};

File: 7-kyu/turn-any-word-into-a-beef-taco/python/solution.py
def tacofy(word):
    INGREDIENTS = {'a': 'beef', 'e': 'beef', 'i': 'beef',
                   'o': 'beef', 'u': 'beef', 't': 'tomato',
                   'l': 'lettuce', 'c': 'cheese', 'g': 'guacamole',
                   's': 'salsa'}
    return ['shell'] + [INGREDIENTS.get(ch) for ch in word.lower() if ch in INGREDIENTS] + ['shell']
